Makefile

```diff
@@ -90,10 +90,7 @@ install-shell-completion:
 	sudo cp scripts/contrib/completion/bash/netctl /etc/bash_completion.d/netctl
 endif
 
-build:
-	make start
-	make ssh-build
-	make stop
+build: start ssh-build stop
```

Comment:
@rchirakk -jX shouldn't be used at all in the context of this repository. It doesn't make sense.